SaaS integration tests not working

### Bug Description
Currently unable to access the Vault secrets to run the SaaS connector integration tests. It looks like the required environment variables are not available after running `nox -s dev -- shell`

- VAULT_NAMESPACE
- VAULT_ADDR
- VAULT_TOKEN

```
======================================================================= ERRORS =======================================================================
___________________________________________________ ERROR at setup of test_stripe_connection_test ____________________________________________________

saas_config = {}

    @pytest.fixture(scope="session")
    def stripe_secrets(saas_config):
        return {
>           "domain": pydash.get(saas_config, "stripe.domain") or secrets["domain"],
            "api_key": pydash.get(saas_config, "stripe.api_key") or secrets["api_key"],
            "payment_types": pydash.get(saas_config, "stripe.payment_types")
            or secrets["payment_types"],
            "page_size": pydash.get(saas_config, "stripe.page_size")
            or secrets["page_size"],
        }
E       TypeError: 'NoneType' object is not subscriptable

tests/ops/fixtures/saas/stripe_fixtures.py:29: TypeError
============================================================== short test summary info ===============================================================
ERROR tests/ops/integration_tests/saas/test_stripe_task.py::test_stripe_connection_test - TypeError: 'NoneType' object is not subscriptable
==================================================== 2 deselected, 2 warnings, 1 error in 11.07s =====================================================
```

#### Steps to Reproduce

1. nox -s dev -- shell
2. pytest tests/ops/integration_tests/saas/test_stripe_task.py -k test_stripe_connection_test

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -27,6 +27,9 @@ services:
       FIDES__REDIS__ENABLED: "True"
       FIDES__TEST_MODE: "True"
       FIDES__USER__ANALYTICS_OPT_OUT: "True"
+      VAULT_ADDR: ${VAULT_ADDR}
+      VAULT_NAMESPACE: ${VAULT_NAMESPACE}
+      VAULT_TOKEN: ${VAULT_TOKEN}
     volumes:
       - type: bind
         source: .

